modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -218,6 +218,19 @@ Any editors using this `highlight_on_focus: true` option, can remove this option
 
 // CCFR here.
 
+=== `convert_unsafe_embeds` editor option is now defaulted to `true`.
+
+In {productname} 6.8.1, https://www.tiny.cloud/docs/tinymce/latest/6.8.1-release-notes/#new-convert_unsafe_embeds-option-that-controls-whether-object-and-embed-elements-will-be-converted-to-more-restrictive-alternatives-namely-img-for-image-mime-types-video-for-video-mime-types-audio-audio-mime-types-or-iframe-for-other-or-unspecified-mime-types[convert_unsafe_embeds] editor option was introduced to support a security issue regarding object tags, which was a legacy method of inserting an external resource, such as a video, a PDF.
```

Comment:
```suggestion
In {productname} 6.8.1, link:https://www.tiny.cloud/docs/tinymce/6/6.8.1-release-notes/#new-convert_unsafe_embeds-option-that-controls-whether-object-and-embed-elements-will-be-converted-to-more-restrictive-alternatives-namely-img-for-image-mime-types-video-for-video-mime-types-audio-audio-mime-types-or-iframe-for-other-or-unspecified-mime-types[convert_unsafe_embeds] editor option was introduced to allow `+object+` and `+embed+` elements to be converted by default to the correct element, respective of the MIME type, automatically when inserted into the editor.
```
It would be better if the `convert_unsafe_embeds` option was a partial page, where it can be included in the `security.adoc` page and the `content-filtering.adoc` page, like I suggested for `sandbox_iframes` [here](https://github.com/tinymce/tinymce-docs/pull/3113/files#r1510723602). Then this can just be an `xref:content-filtering.adoc#convert_unsafe_embeds` link